ui/plugin-lifecycle/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "vcd-sdk-plugin-lifecycle",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "clean": "rimraf dist",
+    "lint-fix": "tslint --fix \"src/**/*.ts\" || exit 0",
+    "build": "gulp build && ncp src/public dist && cd dist && bestzip plugin.zip bundle.js manifest.json i18n.json assets/ && cd ..",
+    "deploy": "python ui_ext_api.py deploy",
+    "remove": "python ui_ext_api.py remove",
+    "removeAllUiExtensions": "python ui_ext_api.py removeAllUiExtensions",
+    "listUiExtensions": "python ui_ext_api.py listUiExtensions"
+  },
+  "license": "BSD-3-Clause",
+  "dependencies": {
+    "@angular/animations": "4.4.4",
+    "@angular/common": "4.4.4",
+    "@angular/compiler": "4.4.4",
+    "@angular/core": "4.4.4",
+    "@angular/forms": "4.4.4",
+    "@angular/http": "4.4.4",
+    "@angular/platform-browser": "4.4.4",
+    "@angular/platform-browser-dynamic": "4.4.4",
+    "@angular/router": "4.4.4",
+    "@ngrx/core": "1.2.0",
+    "@ngrx/effects": "2.0.3",
+    "@ngrx/store": "2.2.2",
+    "@vcd-ui/common": "file:vcd-ui/common",
```

Comment:
this doesn't exist anymore.  yarn and yarn upgrade will error when executed

Reply:
http-transfer-service is another dependency that doesn't exist.

Reply:
I added http-transfer-service in the root directory of the plugin-lifecycle and vcd-ui/common is removed.